src/net/RTCP/RTCPSession.cs

```diff
@@ -161,13 +161,13 @@ public class RTCPSession
         /// <summary>
         /// Event handler for sending RTCP reports.
         /// </summary>
-        internal event Action<SDPMediaTypesEnum, RTCPCompoundPacket> OnReportReadyToSend;
+        public event Action<SDPMediaTypesEnum, RTCPCompoundPacket> OnReportReadyToSend;
 

```

Comment:
This event can already be captured using RTPSession.OnSendReport. Is there any reason to expose this event handler as well?